dasharo-security/tpm2-commands.robot

```diff
@@ -312,6 +313,43 @@ TPMCMD014.001 Encrypt and Decrypt non-rootfs partition (Ubuntu 22.04)
     Execute Command In Terminal    rm -f key seal.* prim.* test-partition
     Execute Linux Tpm2 Tools Command    tpm2_evictcontrol -c 0x81010001
 
+TPMCMD015.001 Encrypt and Decrypt rootfs partition (Ubuntu 22.04)
```

Comment:
@DaniilKl please move this test to tpm-support.robot, I think it is better place than tpm2-commands